src/pages/Messages/index.js

```diff
@@ -217,7 +249,18 @@ const MessagesPage = () => {
                 key={contact.pk}
                 publicKey={contact.pk}
                 subtitle={lastMessage.body}
-                time={DateTime.fromMillis(lastMessage.timestamp).toRelative()}
+                time={(() => {
```

Comment:
Should be moved to a separate `useCallback` wrapped function to avoid creating a new function for every message that's being looped on.

This should make the JS garbage collector trigger less often and result in a smoother, less janky experience for users with hundreds of messages

Reply:
ok